KsApi/models/graphql/adapters/UpdateAccountEnvelope+UpdateAccountEnvelope.Data.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import ReactiveSwift
+
+extension UpdateAccountEnvelope {
+  static func from(_ data: GraphAPI.UpdateUserAccountMutation.Data) -> UpdateAccountEnvelope? {
```

Comment:
Can you add some docs to all the methods here and elsewhere? I'm trying to get into the habit of documenting everything we can moving forward.